embulk-formatter-csv/build.gradle

```diff
@@ -62,6 +68,84 @@ javadocJar {
     from rootProject.file("LICENSE")
 }
 
+publishing {
+    publications {
+         maven(MavenPublication) {
+            afterEvaluate { project ->
+                groupId = "${project.group}"
+                artifactId = "${project.name}"
+            }
+
+            from components.java
+             // javadocJar and sourcesJar are added by java.withJavadocJar() and java.withSourcesJar() above.
+             // See: https://docs.gradle.org/current/javadoc/org/gradle/api/plugins/JavaPluginExtension.html
+
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                packaging "jar"
+
+                afterEvaluate { project ->
+                    name = "${project.name}"
+                    description = "${project.description}"
+                }
+                url = "https://www.embulk.org/"
+
+                licenses {
+                    license {
+                        // http://central.sonatype.org/pages/requirements.html#license-information
+                        name = "The Apache License, Version 2.0"
+                        url = "https://www.apache.org/licenses/LICENSE-2.0.txt"
+                    }
+                }
+
+                developers {
+                    developer {
+                        name = "Sadayuki Furuhashi"
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Hiroyuki Sato"
+                        email = "hiroysato@gmail.com"
+                    }
```

Comment:
@hiroyuki-sato We're going to separate `embulk-standards` plugins as individual plugins, then to release them as independent pure-Java Embulk plugin JARs into Maven Central. In this `embulk-formatter-csv`, we wanted to include you as a developer from the pull request #128. What do you think?

Reply:
Assuming that @hiroyuki-sato is okay with it as he's okay with https://github.com/embulk/embulk/pull/1391#discussion_r586286675

Reply:
Thank you! No problem. Sorry, I've missed this mention.